packages/0x.js/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # CHANGELOG
 
+Pending
+------------------------
+    * Subscribe to ZRX Token Transfer and Approval events when maker token is different
```

Comment:
Add PR number

Reply:
Tell that it's in the orderWatcher. I think that's the first version where it's released